### PR TITLE
General info about Centreon Cloud

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: cloud/getting-started/about-centreon-cloud.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+id: about-centreon-cloud
+title: About Centreon Cloud
+---
+
+## What is Centreon Cloud?
+
+Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure: read our [product description](https://www.centreon.com/centreon-cloud/).
```
Comment (Nohzoh) [inline cloud/getting-started/about-centreon-cloud.md:8]:
```suggestion
Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
```

File: cloud/getting-started/about-centreon-cloud.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## What is the SLA for Centreon Cloud?
+
```
Comment (Nohzoh) [inline cloud/getting-started/about-centreon-cloud.md:11]:
```suggestion

Centreon Cloud is committed to a high service quality with the following guarantees:
```

File: cloud/getting-started/about-centreon-cloud.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Centreon uses [several hosting providers](../security/security.md#hosting-security), mainly in Europe. We are [AWS partners](https://www.centreon.com/partners/centreon-on-aws/).
+
+## How long is my data kept for?
+
+Monitoring data is kept for 13 months: performance data (metrics), downtimes, acknowledgements, comments, logs...
+All other data (e.g. your configuration) is kept until the end of your subscription.
+
+## What security measures does Centreon take?
+
+Everything is explained in our [Centreon Cloud security management](../security/security.md) page.
```
Comment (Nohzoh) [inline cloud/getting-started/about-centreon-cloud.md:29]:
```suggestion
For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).
```

File: cloud/getting-started/about-centreon-cloud.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Fair use policy
+
+Read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
+
+## Maintenance updates and upgrades
+
+* Your central server is updated and upgraded by Centreon, you have nothing to do.
```
Comment (Nohzoh) [inline cloud/getting-started/about-centreon-cloud.md:37]:
```suggestion
* Your central server is fully managed and updated by Centreon. No action is required on your part.
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+id: about-centreon-cloud
+title: À propos de Centreon Cloud
+---
+
+## Qu'est-ce que Centreon Cloud ?
+
+Centreon Cloud est notre plateforme de supervision Centreon installée dans notre infrastructure Cloud : lisez notre [description produit](https://www.centreon.com/fr/centreon-cloud/).
```
Comment (Nohzoh) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:8]:
```suggestion
Centreon Cloud est notre plateforme de supervision Centreon installée dans notre infrastructure Cloud. Pour plus d'informations, consultez notre [page de description du produit](https://www.centreon.com/fr/centreon-cloud/).
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Quel SLA pour Centreon Cloud ?
+
```
Comment (Nohzoh) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:11]:
```suggestion

Centreon Cloud s'engage à offrir un service de haute qualité avec les garanties suivantes :

```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+Centreon utilise [plusieurs hébergeurs](../security/security.md#sécurité-de-lhébergement), principalement situés en Europe. Nous sommes [partenaires AWS](https://www.centreon.com/fr/partenaires/centreon-et-aws/).
+
+## Combien de temps sont gardées mes données ?
+
+Les données de supervision sont gardées 13 mois : données de performance data (métriques), plages de maintenance, acquittements, commentaires, logs...
+Toutes les autres données (par exemple la configuration) sont gardées jusqu'à la fin de votre souscription.
+
+## Quelles mesures de sécurité Centreon prend-il ?
+
+Tout est expliqué dans notre page [Centreon Cloud - Management de la sécurité](../security/security.md).
```
Comment (Nohzoh) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:29]:
```suggestion
Pour une vue d'ensemble complète de nos mesures de sécurité, visitez notre page dédiée à la [gestion de la sécurité de Centreon Cloud](../security/security.md).
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Accord d'usage non abusif (Fair Use Policy)
+
+Consultez l'[accord d'usage non-abusif](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
```
Comment (Nohzoh) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:33]:
```suggestion
Pour comprendre nos limitations en matière d'utilisation des ressources, consultez notre [accord d'usage non-abusif](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
```

**Review by Nohzoh — DISMISSED** [review]

File: cloud/getting-started/about-centreon-cloud.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+title: About Centreon Cloud
+---
+
+## What is Centreon Cloud?
+
+Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
+
+## What is the SLA for Centreon Cloud?
+
+Centreon Cloud is committed to a high service quality with the following guarantees:
```
Comment (cg-tw) [inline cloud/getting-started/about-centreon-cloud.md:12]:
```suggestion
Centreon Cloud is committed to a high quality of service with the following guarantees:
```

File: cloud/getting-started/about-centreon-cloud.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+Monitoring data is kept for 13 months: performance data (metrics), downtimes, acknowledgements, comments, logs...
+All other data (e.g. your configuration) is kept until the end of your subscription.
+
+## What security measures does Centreon take?
+
+For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).
+
+## Fair use policy
+
+Read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
```
Comment (cg-tw) [inline cloud/getting-started/about-centreon-cloud.md:35]:
```suggestion
To understand our limitations in terms of resource use, read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
```

**Review by Nohzoh — APPROVED** [review]

File: cloud/getting-started/about-centreon-cloud.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
+---
+id: about-centreon-cloud
+title: About Centreon Cloud
+---
+
+## What is Centreon Cloud?
+
+Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
```
Comment (rchauvel) [inline cloud/getting-started/about-centreon-cloud.md:8]: Should rather be: 
Centreon Cloud is the Centreon monitoring platform delivered as a service and hosted and managed by Centreon from our Cloud infrastructure.

File: cloud/getting-started/about-centreon-cloud.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).
+
+## Fair use policy
+
+To understand our limitations in terms of resource use, read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
+
+## Maintenance updates and upgrades
+
+* Your central server is fully managed and updated by Centreon. No action is required on your part.
```
Comment (rchauvel) [inline cloud/getting-started/about-centreon-cloud.md:39]: Should rather be:
The web application, data ingestion, transformation, and storage are fully managed [...]

File: cloud/getting-started/about-centreon-cloud.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+* Your pollers are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).
```
Comment (rchauvel) [inline cloud/getting-started/about-centreon-cloud.md:40]: Can you add a link to the glossary for "pollers"?
Also "it is your responsibility to install and keep them up-to-date"

Comment (cg-tw) [inline cloud/getting-started/about-centreon-cloud.md:40]:
```suggestion
* Your [pollers](../resources/glossary.md#poller) are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
+---
+id: about-centreon-cloud
+title: À propos de Centreon Cloud
+---
+
+## Qu'est-ce que Centreon Cloud ?
+
+Centreon Cloud est notre plateforme de supervision Centreon installée dans notre infrastructure Cloud. Pour plus d'informations, consultez notre [page de description du produit](https://www.centreon.com/fr/centreon-cloud/).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:8]:
```suggestion
Centreon Cloud est la plateforme de supervision Centreon fournie en tant que service, hébergée et gérée par Centreon depuis notre infrastructure Cloud. Pour plus d'informations, consultez notre [page de description du produit](https://www.centreon.com/fr/centreon-cloud/).
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+
+Pour une vue d'ensemble complète de nos mesures de sécurité, visitez notre page dédiée à la [gestion de la sécurité de Centreon Cloud](../security/security.md).
+
+## Accord d'usage non abusif (Fair Use Policy)
+
+Pour comprendre nos limitations en matière d'utilisation des ressources, consultez notre [accord d'usage non-abusif](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
+
+## Maintenance : mises à jour et montées de version
+
+* Votre serveur central est mis à jour et monté de version par Centreon, vous n'avez rien à faire.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/getting-started/about-centreon-cloud.md:39]:
```suggestion
* La plateforme Centreon Cloud (application web, ingestion des données, transformation, stockage...) est entièrement gérée et mise à jour par Centreon. Aucune action de votre part n'est requise.
```

**Review by Nohzoh — APPROVED** [review]

**Review by rchauvel — APPROVED** [review]

**Pull request merged**